### ISSUE TITLE
Question: why expect the CA cert in configmap, instead of as a property 'ca.crt' in the secret containing the TLS.crt and TLS.key?

### ISSUE THREAD

**Issue by kylos101** [mongodb/mongodb-kubernetes-operator#114]
Hi there,

First, may  I say it is awesome that you're all spinning up a MongoDB community operator!

Second, I noticed the CA cert is expected to be in the ConfigMap, as `ca.crt` per #75 and #81. May I ask why you look for the `ca.crt` in the ConfigMap, and not the Secret? 

As background:
My understanding is, when cert-manager provisions a Certificate, and persists it to the k8s cluster within a Secret, it stores the `ca.crt` as a property in the related secret, too. [I just submitted this PR to update the cert-manager docs, but wanted to give you all a heads up.](https://github.com/cert-manager/website/pull/256)

In other words, if you leave the reference to the `ca.crt` in the ConfigMap (instead of moving it to the same secret where the tls.crt, and tls.key live), it'll create more work for folks that use cert-manager, because they'll have to copy the `ca.crt`'s value from the Secret to a ConfigMap. 

It's entirely possible I'm missing something here, but wanted to throw it out there for discussion. Thanks a lot for your help!

**Comment by kylos101** [comment]
Here are the contents of a secret provisioned by a cert-manager Certificate:

![image](https://user-images.githubusercontent.com/1272076/87884732-7f118a00-c9de-11ea-8040-13f74d08656c.png)

**Comment by fabianlindfors** [comment]
Hi, Kyle!

As the comment on your PR indicates, not all cert-manager issuers support the `ca.crt` field and hence we opted to accept it separately. Our main goal is to support the `kubernetes.io/tls` Secret format which only requires `tls.crt` and `tls.key`.

In the future we might add support for passing the CA certificate in the Secret as it would definitely be convenient. It will not be part of our first release with TLS though.

**Comment by kylos101** [comment]
Hi, @fabianlindfors,

Mind = blown! 👍 

Thank you very much for reaching out to clarify, I appreciate it.
